### ISSUE TITLE
Failed to load module ngSanitize

### ISSUE THREAD

**Issue by radarlui** [jbossdemocentral/brms-coolstore-demo-angular-patternfly#5]
I tried running this demo without using Docker and I got a blank page when I opened the brms-coolstore-demo page. I debugged the javascript and found that it fails to load the module ngSanitize. Thus I changed as follow and it is working now.

app.js - Changed to: var module = angular.module('app', [ 'ngRoute', 'patternfly', 'ngSanitize' ]), ...
index.html: - Added: <script src="vendor/angular-sanitize/angular-sanitize.min.js"></script>

**Comment by jamesfalkner** [comment]
Yeah, we are tracking 'latest' PatternFly (not exactly a best practice) and the charts submodule added a new dependency on ngSanitize. Will fix up in a bit. Thanks for noticing!
